src/Microsoft.AspNetCore.Server.Testing/Deployers/RemoteWindowsDeployer/RemotePSSessionHelper.ps1

```diff
@@ -10,6 +10,9 @@ param(
 	[string]$accountPassword,
 
 	[Parameter(Mandatory=$true)]
+	[string]$dotnetRuntimePath,
```

Comment:
I don't think this should be a mandatory parameter. We don't need this for standalone apps.